rosdep/python.yaml

```diff
@@ -96,7 +96,7 @@ python-argparse:
     squeeze: [python-argparse]
     wheezy: [python-argparse]
   fedora: [python]
-  gentoo: [virtual/python-argparse]
+  gentoo: [dev-python/argparse]
```

Comment:
Gentoo removed this definition, so I have added the ebuild under the name dev-python/argparse.